### PR TITLE
Retrieve user's profile from auth (#1716) and expose feature info (#1730)

### PR DESCRIPTION
**Pull request by xcoulon**
Also
- retrieve feature content from unleash
- merge feature controllers into a single one

Fixes #openshiftio/openshift.io/1716
Fixes #openshiftio/openshift.io/1730

Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

**Comment by xcoulon** [comment]
@corinnekrych this PR must not be merged until https://github.com/openshiftio/openshift.io/issues/1717 is fixed

File: main.go [hunk 1]
```diff
@@ -14,7 +14,6 @@ import (
 	witmiddleware "github.com/fabric8-services/fabric8-wit/goamiddleware"
 	"github.com/fabric8-services/fabric8-wit/log"
 	"github.com/goadesign/goa"
-	"github.com/goadesign/goa/logging/logrus"
```
Comment (corinnekrych) [inline main.go]: why remove it in the import but not line 40?

File: glide.lock [hunk 2]
```diff
@@ -38,13 +48,13 @@ imports:
...
   - design/apidsl
   - dslengine
+  - encoding/form
   - goagen
   - goagen/codegen
   - goagen/gen_app
   - goagen/gen_controller
   - goagen/utils
   - goatest
-  - logging/logrus
```
Comment (corinnekrych) [inline glide.lock]: do we really want to remove logrus?

Comment (xcoulon) [reply glide.lock]: we can use the logger wrapper provided by `fabric8-auth`

File: main.go (hunk 1 above)
Comment (xcoulon) [reply main.go]: not sure why it was removed, but it's been restored with  `goalogrus` alias now.

**Pull request merged**